8_August.sql

```diff
@@ -0,0 +1,58 @@
+assignment 1 : 
+create 'student_db' database
+create following tables in database with proper data types
+student: id (PK), name, address, phone, email, age, schoolId (FK)
+school: id (PK), name, address, principal, phone
+create relationship as shown in the table schema
+add some dummy data
+
+assignment 1 Answer :
+
+create database student_db;
```

Comment:
Always use caps for command for example 

CREATE DATABASE student_db

SELECT name, address FROM student